core/settings/_breakpoints.scss

```diff
@@ -59,7 +59,7 @@ $breakpoints: (
 
 /**
  * A global setting to define the common breakpoint(s) for Scally and your
- * project. This is mainly used by the Generate at breakpoints mixin.
+ * project. This is mainly used by the `generate-at-breakpoints()` mixin.
```

Comment:
You've introduced a new format for mixin names when inside comments. I actually prefer this format but my OCD can't stand the inconsistency this has introduced. Fear not I shall apply this new format, [see](https://github.com/chris-pearce/scally/issues/128). Worth keeping that in mind though i.e. if there's a convention already lets be consistent and stick to it, and if you don't like it then raise an issue :+1:.